tutorials/polkadot-sdk/parachains/zero-to-hero/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Zero To Hero
+description: A comprehensive guide for developers to build, test, and deploy custom pallets and runtimes, leveraging the full potential of the Polkadot SDK.
+hide: 
+    - feedback
+template: index-page.html
+---
+
+# Zero To Hero Tutorials
+
+The **Zero To Hero Tutorials** provide developers with a step-by-step guide to building, testing, and deploying custom pallets and runtimes using the Polkadot SDK. These tutorials are designed to help you gain hands-on experience and understand the core concepts necessary to create efficient and scalable blockchains.  
+
```

Comment:
Please add a "!!! tip" admonition with text:

Start with the "Set Up a Template" guide to get the most from this section. As you complete each guide, look for "Where to Go Next" to move to the following guide in the series.

Reply:
fixed on [22c46b0](https://github.com/polkadot-developers/polkadot-docs/pull/254/commits/22c46b0cb19fc1521d8f2f80be1b050e20326b02)